app/models/game-participant.js

```diff
@@ -30,19 +30,32 @@ export default DS.Model.extend({
   /** @property {String} Convenience property for computed dependencies. */
   userId: Ember.computed.alias('user.id'),
 
-  gameCards: Ember.computed('gameCardsRaw', function (key, value) {
-    // setter
-    if (arguments.length > 1) {
+  gameCards: Ember.computed('gameCardsRaw', {
+    get() {
```

Comment:
One day you'll have to let me know how you keep up with the latest ember stuff when you don't even use it at work